org.lemonade_emu.lemonade.json

```diff
@@ -0,0 +1,172 @@
+{
+    "app-id": "org.lemonade_emu.lemonade",
```

Comment:
does upstream really control `lemonade-emu.org`. I see nothing there.

Reply:
@hfiguiere 
now we changed it to io.github.lemonade-emu.Lemonade as our website https://lemonade-emu.github.io/